server/apiv2/handlers/tso_keyspace_group.go

```diff
@@ -237,6 +239,68 @@ func FinishSplitKeyspaceByID(c *gin.Context) {
 	c.JSON(http.StatusOK, nil)
 }
 
+// MergeKeyspaceGroupsParams defines the params for merging the keyspace groups.
+type MergeKeyspaceGroupsParams struct {
+	MergeList []uint32 `json:"merge-list"`
+}
+
+// MergeKeyspaceGroups merges the keyspace groups in the merge list into the target keyspace group.
+func MergeKeyspaceGroups(c *gin.Context) {
+	id, err := validateKeyspaceGroupID(c)
```

Comment:
Do we need to check whether their user kind is the same?

Reply:
I don't think it's necessary for the time being. At present, as a split rollback, we should make it as easy as possible.